src/main/kotlin/me/serce/solidity/lang/stubs/impl.kt

```diff
@@ -233,4 +234,28 @@ class SolEventDefStub(
   }
 }
 
+class SolImportPathDefStub(
+  parent: StubElement<*>?,
+  elementType: IStubElementType<*, *>,
+  override val name: String?,
+  val path: String?
+) : StubBase<SolImportPathImpl>(parent, elementType), SolNamedStub {
+
+  object Type : SolStubElementType<SolImportPathDefStub, SolImportPathImpl>("IMPORT_PATH") {
+    override fun deserialize(dataStream: StubInputStream, parentStub: StubElement<*>?) =
+      SolImportPathDefStub(parentStub, this, dataStream.readNameAsString(), dataStream.readUTFFast())
+
+    override fun serialize(stub: SolImportPathDefStub, dataStream: StubOutputStream) = with(dataStream) {
+      writeName(stub.name)
+      writeUTFFast(stub.path ?: "")
+    }
+
+    override fun createPsi(stub: SolImportPathDefStub) = SolImportPathImpl(stub, this)
+
+    override fun createStub(psi: SolImportPathImpl, parentStub: StubElement<*>?) = SolImportPathDefStub(parentStub, this, psi.name, psi.text)
+
+    override fun indexStub(stub: SolImportPathDefStub, sink: IndexSink) = sink.indexImportPathDef(stub)
+  }
+}
+
```

Comment:
The stub tree was changed, so this counter needs to be bumped:
```
me.serce.solidity.lang.stubs.SolidityFileStub.Type#getStubVersion
```

Reply:
done